### ISSUE TITLE
Compatibility with the latest version of all dependencies

### ISSUE THREAD

**Issue by benjaminhuo** [OpenFunction/OpenFunction#260]
**Description**
Knative, Tekton, Shipwright, Dapr, KEDA all released new versions, we need to check the compatibility of OpenFunction with these.
And possibly need to add or update the compatibility matrix.

**Additional context**
- [ ] [Knative Eventing v1.3.0](https://github.com/knative/serving/releases/tag/knative-v1.3.0)
- [ ] [Tekton pipeline v0.34.1](https://github.com/tektoncd/pipeline/releases/tag/v0.34.1)
- [ ] [Shpwright v0.8.0](https://github.com/shipwright-io/build/releases/tag/v0.8.0)
- [ ] [KEDA v2.6.1](https://github.com/kedacore/keda/releases/tag/v2.6.1)
- [ ] [Dapr v1.6.x+](https://github.com/dapr/dapr/releases)

**Comment by benjaminhuo** [comment]
Dapr and Keda => @tpiperatgod 
Knative Serving => @benjaminhuo 
Tekton & Shipwright => @wanjunlei

**Comment by tpiperatgod** [comment]
I have tested OpenFunction with Dapr 1.6.1 and KEDA v2.6.1 and it passed.

Please note that in Dapr 1.6.1, you may need to make some adjustments to the Components to achieve the effect in 1.6.1- scenarios. For example, when using `pubsub.kafka`, you may need to set the `disableTls` property to `true` to access a Kafka cluster without TLS enabled, which is absent in the 1.6.1- scenarios.

**Comment by webup** [comment]
All of the framework owners should pay special attention to the Dapr upgrade, you'd better check these 3 major versions 1.6 / 1.7 / 1.8 for compatibility, for example, I just found 1.8.0 broke my e2e tests. T.T

**Comment by benjaminhuo** [comment]
yep, we should pay more attention to Dapr @tpiperatgod @wanjunlei @wrongerror @lizzzcai , especially functions-frameworks
